libsplinter/src/rest_api/mod.rs

```diff
@@ -32,7 +32,7 @@
 //!     fn resources(&self) -> Vec<Resource> {
 //!         let name = self.name.clone();
 //!
-//!         vec![Resource::new(Method::Get, "/index", move |r, p| {
```

Comment:
I would suggest that we deprecate the old `new` and make the new way `build`

Reply:
Done